src/sentry/south_migrations/0293_auto__del_unique_release_project_id_version.py

```diff
@@ -0,0 +1,711 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Removing unique constraint on 'Release', fields ['project_id', 'version']
+        db.delete_unique(u'sentry_release', ['project_id', 'version'])
```

Comment:
meh, I guess you can't. According to Postgres docs, you can't do CONCURRENTLY here.

> There are several caveats to be aware of when using this option. Only one index name can be specified, and the CASCADE option is not supported. (Thus, an index that supports a UNIQUE or PRIMARY KEY constraint cannot be dropped this way.) Also, regular DROP INDEX commands can be performed within a transaction block, but DROP INDEX CONCURRENTLY cannot.

This will be fine.